<?xml version="1.0"?>
<launch>
  
  <arg name="launch-prefix"               default=""/>
  <arg name="config_files_folder"         default="$(find point_cloud_processor)/config"/>

  <node pkg="point_cloud_processor" type="point_cloud_processor_node" name="point_cloud_processor" output="screen" launch-prefix="$(arg launch-prefix)">
    <param name="config_files_folder"       value="$(arg config_files_folder)"/>
    <rosparam command="load" file="$(arg config_files_folder)/sensor_streams_example.yaml"/>
  </node>

</launch>
